linkml_model/model/schema/meta.yaml

```diff
@@ -2410,3 +2534,33 @@ enums:
         meaning: skos:broaderMatch
       NARROW_SYNONYM:
         meaning: skos:narrowerMatch
+
+  # I'm not really sure if this is valid/correct, but I wanted to give a quick and dirty example. This is based on: https://build.fhir.org/valueset-example-intensional.json.html - joeflack4 2022/05/27
+  serum_cholesterol_enum:
+    # `description`: I didn't see this as a valid `slot` for `enum_expression`. Should this be here?
```

Comment:
`description`: I didn't see this as a valid `slot` for `enum_expression`. Should this be here?